src/Components/Layout/LayoutPublic.js

```diff
@@ -0,0 +1,15 @@
+import Header from './Header/PublicHeader'
+import Footer from '../UI/Footer/Footer'
+import './LayoutPublic.css'
+
+const LayoutPublic = ({ children }) => {
+	return (
+		<div>
+			<Header />
+			<div className="layout-container">{children}</div>
+			<Footer />
+		</div>
```

Comment:
Acá el div no es necesario, usaría react fragment
```suggestion
		<>
			<Header />
			<div className="layout-container">{children}</div>
			<Footer />
		<>
```